src/patterns/ViewingDetails/ViewingDetails.tsx

```diff
@@ -0,0 +1,175 @@
+   */
+  centerAlignText?: boolean;
+}
+/**
+ * ## Overview
+ *
+ * A simple component for displaying viewing details such as session times, viewing times, location, and a map link.
+ *
+ * [Figma Link](https://www.figma.com/design/H1kCh6MXU8jasYbQuCbyBt/Calendar?node-id=6-17&p=f&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-viewingdetails--overview)
+ *
+ */
+const ViewingDetails = forwardRef<HTMLDivElement, ViewingDetailsProps>(
+  (
+    {
+      className,
+      children,
+      label,
+      sessionTimesLabel,
+      sessionTimes,
+      viewingTimes,
+      location,
+      mapLink,
+      centerAlignText = false,
+      onClose,
+      linkElement: Component = Link,
+      ...props
+    }: ViewingDetailsProps,
+    ref,
+  ) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'ViewingDetails');
+
+    return (
+      <div {...commonProps} {...props} className={classnames(baseClassName, className)} ref={ref}>
+        <div className={`${baseClassName}__content`}>
+          {children && <div className={`${baseClassName}__children`}>{children}</div>}
+
+          {sessionTimesLabel && (
+            <Text variant={TextVariants.heading4} className={centerAlignText ? `${baseClassName}__center-align` : ''}>
+              {sessionTimesLabel}
+            </Text>
+          )}
+          {sessionTimes &&
+            sessionTimes.length > 0 &&
+            sessionTimes.map((session) => (
+              <div key={`${session.sessionLabel ?? ''}-${session.sessionTime ?? ''}`}>
+                {session.sessionLabel && (
+                  <Text
+                    variant={TextVariants.heading5}
+                    className={classnames(`${baseClassName}__label`, {
+                      [`${baseClassName}__center-align`]: centerAlignText,
+                    })}
+                  >
+                    {session.sessionLabel}
+                  </Text>
+                )}
+                {session.sessionTime && (
+                  <Text
+                    variant={TextVariants.body2}
+                    className={classnames(`${baseClassName}__text`, {
+                      [`${baseClassName}__center-align`]: centerAlignText,
+                    })}
+                  >
+                    {session.sessionTime}
+                  </Text>
+                )}
+              </div>
+            ))}
+          {label && (
+            <Text variant={TextVariants.heading4} className={centerAlignText ? `${baseClassName}__center-align` : ''}>
+              {label}
+            </Text>
+          )}
+          {viewingTimes &&
+            viewingTimes.length > 0 &&
+            viewingTimes.map((time) => (
+              <Text
+                key={time}
+                variant={TextVariants.body2}
+                className={classnames(`${baseClassName}__label`, {
+                  [`${baseClassName}__center-align`]: centerAlignText,
```

Comment:
Not a blocker but I would probably put this on the containing element so you didnt have to add this logic to each item.